modules/openapi-generator/src/main/resources/go-server/Dockerfile.mustache

```
@@ -1,10 +1,12 @@
-FROM golang:1.10 AS build
+FROM golang:1.19 AS build
 WORKDIR /go/src
 COPY {{sourceFolder}} ./{{sourceFolder}}
 COPY main.go .
+COPY go.sum .
+COPY go.mod .
 
 ENV CGO_ENABLED=0
-RUN go get -d -v ./...
+RUN go install
 
 RUN go build -a -installsuffix cgo -o {{packageName}} .
```

```suggestion
RUN go build -o {{packageName}} .
```

this part is not needed anymore, but fine to leave it if you prefer